### ISSUE TITLE
Issue while building intel-parallel-studio @ 2016.03

### ISSUE THREAD

**Issue by pramodskumbhar** [spack/spack#5648]
I am seeing : 

```
+ spack install --show-log-on-error intel-parallel-studio@professional.2016.3+advisor+inspector+itac+vtune %gcc@4.8.4
==> Installing intel-parallel-studio
==> Fetching file:///gpfs-path//SPACK_HOME/mirrors/compiler/intel-parallel-studio/intel-parallel-studio-professional.2016.3.tgz
==> Staging archive: /gpfs-path/SPACK_HOME/spack/var/spack/stage/intel-parallel-studio-professional.2016.3-fdcd2krvyzfy2lafx5uobzyx23amhdsx/parallel_studio_xe_2016_update3.tgz
==> Created stage in /gpfs-path/SPACK_HOME/spack/var/spack/stage/intel-parallel-studio-professional.2016.3-fdcd2krvyzfy2lafx5uobzyx23amhdsx
==> No patches needed for intel-parallel-studio
==> Building intel-parallel-studio [IntelPackage]
==> Found already existing license /gpfs-path/SPACK_HOME/spack/etc/spack/licenses/intel/license.lic
==> Executing phase: 'configure'
==> Executing phase: 'install'
==> Error: IOError: [Errno 2] No such file or directory: '/gpfs-path/install/compilers/install/linux-rhel6-x86_64/gcc-4.8.4/intel-parallel-studio-professional.2016.3/parallel_studio_xe_2016.3.067/bin/psxevars.sh'
IOError: IOError: [Errno 2] No such file or directory: '/gpfs-path/install/compilers/install/linux-rhel6-x86_64/gcc-4.8.4/intel-parallel-studio-professional.2016.3/parallel_studio_xe_2016.3.067/bin/psxevars.sh'

/gpfs-path/SPACK_HOME/spack/var/spack/repos/builtin/packages/intel-parallel-studio/package.py:428, in fix_psxevars:
     7     
     8             filter_file('^SCRIPTPATH=.*', 'SCRIPTPATH={0}'.format(self.prefix),
     9                         os.path.join(bindir, 'psxevars.sh'),
  >> 10                        os.path.join(bindir, 'psxevars.csh'))

```

2017.04 is installing fine. Have to check files/directory structure of installed prefix.

**Comment by weijianwen** [comment]
I reported a duplicate issue in #6522 , which should be fixed by #6523 .

**Comment by scheibelp** [comment]
Closing this on account of merging #6523. Feel free to reopen if the issue persists.
